### DISCUSSION TITLE
The byte representation of the transaction

### DISCUSSION THREAD

**Discussion by advortsov** [otter-sec/anchor#1715] [Q&A]
Hello,

Where can I get a byte representation of a transaction that is generated by the client side? 
And where can I read the rules by which the anchor generates a byte representation of the transaction?

**Comment by kehiy** [comment]
hello @advortsov 
To get the byte representation of a transaction generated by the client side using Rust, Anchor, and Solana, you can use the `processor::seeded::TransactionSerde` module from the `anchor-lang` crate.
